src/v/raft/configuration.h

```diff
@@ -18,60 +18,99 @@
 
 #include <algorithm>
 #include <numeric>
+#include <optional>
 #include <type_traits>
 
 namespace raft {
 
+class node_uid {
```

Comment:
what's `uid` i'd imagine this to be `node_rev_id` or

Reply:
or call it vnode or rnode or something. this is a logical / virtual node, right?

Reply:
yes, let me rename it to `vnode`